engineering/passports/id/DESIGN_DOC.md

```diff
@@ -59,20 +65,28 @@ Each user can have an arbitrary number of passports associated with them but onl
 
 A basic interface reminiscent of other current sign in platforms will ask the user if they want to approve the connection with the target app and list the scopes the app is requesting after they verify ownership of their passport. Passport ownership is verified by entering the passport number then scanning the passport with a phone to contact the server and allow the sign in.
 
+The user management system will allow a user to enroll/unenroll from 2FA and manage aspects of their profile as well as adding their own clients to work with ID. The admin side will allow arbitrary user operations (enable/disable, see 2FA, etc).
+
+The metrics and events system, groups, and (future) policies will be customizable from the admin dashboard.
+
 ### Deployment
 
 Users can integrate with ID using the example repo as a base.
 
 ## Milestones
 
-- Full OAuth support within a month
-- Easy integration instructions/example within a week afterwards
+- User management
+- Arbitrary user 2FA enrollment
+- Advanced client management system
+- Metrics and events active
+- Groups API
+- Future: Policies system
 
 ## Additional Considerations
 
 ### Security
 
-Passports are not secure since they just use basic NFC tags. For most things this is fine but for high-security apps it is possible to swipe an administrator’s passport and scan it when asked. Therefore administrators will be required to have two-factor authentication (TOTP specifically) enabled on their account for all integrations.
+Passports on their own are not secure since they just use basic NFC tags. For most applications this is fine but for high-security apps it is possible to swipe an administrator’s passport and scan it when asked. Therefore apps can require two-factor authentication (TOTP specifically) if they necessitate secure authentication.
```

Comment:
2FA should also support WebAuthn, as that is a more modern standard and is significantly nicer to use. Most modern services support both.

Additionally, 2FA implementation should be mentioned in more detail in the goals section